CMakeLists.txt

```diff
@@ -543,161 +543,19 @@ if(CMAKE_CXX_COMPILER MATCHES ".*clang\\+\\+")
 # Enable tidy on hip
 elseif(MIOPEN_BACKEND STREQUAL "HIP" OR MIOPEN_BACKEND STREQUAL "HIPNOGPU")
     set(MIOPEN_TIDY_ERRORS ALL)
-
 endif()
 
 include(ClangTidy)
 enable_clang_tidy(
     CHECKS
-        *
-        -abseil-*
```

Comment:
[Notice] Before this change, is was potentially possible to switch off and on some checks depending on HIP version. Well, this possibility is kept but will require a little bit more work.